app/assets/javascripts/app/discussions/discussions.module.js

```diff
@@ -31,7 +31,28 @@
       })
       .state('discussions.new', {
         url: '/new',
-        templateUrl: 'discussions/new.html'
+        templateUrl: 'discussions/templates/discussion.html',
+        controller: 'NewDiscussionCtrl as ctrl'
+      })
+      .state('discussions.new.title', {
```

Comment:
I don't like these names, `title` and `body` don't seem like good names. What do you think about `step1` and `step2` for instance?